src/raft/uv_metadata.c

```diff
@@ -146,6 +143,7 @@ int uvMetadataLoad(const char *dir, struct uvMetadata *metadata, char *errmsg)
 	/* Check the versions. */
 	if (metadata1.version == 0 && metadata2.version == 0) {
 		/* Neither metadata file exists: have a brand new server. */
+		metadata->format_version = 0;
```

Comment:
[Nit]: If we want to set all fields to zero we could do `metadata = {0}` and not have to change this code when we add more fields.

Reply:
Are you sure that doing `metadata = {}` is equivalent to initializing all the fields to 0? I thought that you needed the explicit `= {0}` for that, and without it, it means to take the uninitialized memory.